### DISCUSSION TITLE
Language Card Showing languages from private Repos?

### DISCUSSION THREAD

**Discussion by PeterM-13** [anuraghazra/github-readme-stats#2346] [Q&A] [answered]
I only have a few public repos, but many private ones written in various languages. 
So I was wondering if it was possible for the Languages Card to display the languages used in these private repos (as it doesn't atm)?
Is this at all possible, or am I dreaming?

**Comment by Qubik65536** [comment, marked as answer]
You should deploy your own instance of github-readme-stats on Vercel to be able to do that. Check #814
